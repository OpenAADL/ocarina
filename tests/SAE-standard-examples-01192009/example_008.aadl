-- test 4.8 Annex subclauses draft-01192009

-- add for test
package Test
public 
-- end add

thread Collect_Samples
features
Input_Sample : in data port Sampling::Sample;
Output_Average : out data port Sampling::Sample;
annex Error_Model {**
Model => Transient_Fault_Model;
Occurrence => 10e-4 poisson applies to Transient_Fault;
**};
end Collect_Samples;

-- add for test
end Test;
-- end add