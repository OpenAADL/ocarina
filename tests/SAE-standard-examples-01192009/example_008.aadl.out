example_008.aadl:12:19: warning: no support provided for annex language error_model
example_008.aadl:10:29: Sampling::Sample (entity reference) qualified reference name not found in 'with' statements of Test (package specification) 
example_008.aadl:11:32: Sampling::Sample (entity reference) qualified reference name not found in 'with' statements of Test (package specification) 
Cannot analyze AADL specifications
